### DISCUSSION TITLE
다운로드했을 때 모서리가 둥글어요

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#121] [Ideas] [answered]
### 주제

Features

### Body

다운로드했을 때 모서리가 둥글어요!
제작하는 동안 미리보기는 둥글어보여도, 다운로드 되었을 때에는 직각이면 더 **범용성**있지 않을까요!🤗

`html-to-img` 로 변환되는 영역의 상위 태그에 `border-radius` 속성을 주고, `overflow hidden` 으로 처리하면 해결이 되지 않을지 끄적여봅니다!

### BDD TASK

- [ ] 이미지 다운로드 버튼을 누른다 > 모서리가 둥글지 않은 이미지가 저장된다

**Comment by caseBread** [comment]
저도 직각이면 좋겠습니다! 

체다 의견대로 html-to-img 변환 과정에서 css 설정 살짝(?) 만져주면  되지않을까 싶습니다

**Comment by da-in** [comment]
제가 마저 작업 진행하겠습니다.🤣

**Comment by da-in** [comment, marked as answer]
수정되었습니다.
- #136
